packages/renderer/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@muban/storybook",
+  "version": "8.0.0",
+  "description": "Storybook for Muban: View and Test Muban components.",
+  "keywords": [
+    "muban",
+    "storybook",
+    "renderer"
+  ],
+  "author": "Arjan van Wijk",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/@mubanjs/muban-storybook.git"
+  },
+  "bugs": {
+    "url": "https://github.com/@mubanjs/muban-storybook/issues"
+  },
+  "homepage": "https://github.com/mubanjs/muban-storybook/tree/main/packages/renderer#readme",
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": "./dist/index.js",
+      "types": "./dist/index.d.ts"
+    },
+    "./preview": {
+      "import": "./dist/config.js",
+      "types": "./dist/config.d.ts"
+    },
+    "./package.json": "./package.json"
+  },
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist/**/*",
+    "README.md",
+    "*.js",
+    "*.d.ts"
+  ],
+  "scripts": {
+    "dev": "npm run build -- --watch",
+    "build": "tsc -p ./tsconfig.build.json",
+    "test": "jest --config ./jest.config.ts ./src/",
+    "typecheck": "tsc --project tsconfig.json --noEmit --noUnusedLocals",
+    "lint": "npm-run-all lint:*",
+    "lint:eslint": "eslint --ext .js,.jsx,.ts,.tsx,.mjs,.cjs,.mts,.cts . --cache --cache-location node_modules/.cache/.eslintcache",
+    "fix": "npm-run-all fix:*",
+    "fix:eslint": "npm run lint:eslint -- --fix",
+    "prepublishOnly": "npm run build"
+  },
+  "dependencies": {
+    "@muban/muban": "^1.0.0-alpha.35",
+    "@muban/template": "^1.0.0-alpha.4",
+    "@storybook/core-client": "^7.0.7",
+    "@storybook/docs-tools": "^7.0.7",
+    "@storybook/global": "^5.0.0",
+    "@storybook/preview-api": "^7.0.7",
+    "@storybook/types": "^7.0.7",
+    "qs": "^6.11.1",
```

Comment:
You're using this package just once and I think you can change the implementation of that code so that it's not necessary to use this package at all.

```ts
  const fetchUrl = getUrl(
    url,
    path,
    new URLSearchParams([
      ...Object.entires(storyContext.globals),
      ...Object.entries(parameters),
    ])
  );
```

```ts
> new URLSearchParams([['kaas', 'ham'], ['milk', 'water']]).toString()
> kaas=ham&melk=water
```

Reply:
URLSearchParams doesn't work with nested structures unfortunately.